### ISSUE TITLE
Cannot read property 'textChanges' of undefined

### ISSUE THREAD

**Issue by SUCHMOKUO** [simonhaenisch/prettier-plugin-organize-imports#1]
Hi!
The program sometimes crashes, could you fix it? :)
```
C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier\index.js:9755
    throw error.stack;
    ^
TypeError: Cannot read property 'textChanges' of undefined
    at Object.organizeImports [as preprocess] (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier-plugin-organize-imports\index.js:17:43)
    at Object.parse (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier\index.js:9734:21)
    at coreFormat (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier\index.js:13252:23)
    at format (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier\index.js:13510:73)
    at formatWithCursor (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier\index.js:13526:12)
    at C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier\index.js:44207:15
    at Object.format (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\prettier\index.js:44226:12)
    at _default (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\pretty-quick\dist\processFiles.js:42:29)
    at _default (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\pretty-quick\dist\index.js:58:29)
    at Object.<anonymous> (C:\Users\mokuo\Documents\Projects\scrum-board-front-end\node_modules\pretty-quick\bin\pretty-quick.js:12:27)  
```

**Comment by simonhaenisch** [comment]
Thanks for the issue. Just wondering, could you share the file that this doesn't work on with me? I'd like to know how to reproduce this.

**Comment by SUCHMOKUO** [comment]
OK. But this bug occurs not very frequently, I'll report to this issue when it happens next time.

**Comment by simonhaenisch** [comment]
Ok thanks. Does it happen when there's a syntax error in a file by any chance?

**Comment by SUCHMOKUO** [comment]
Maybe no. Because every time it happens, there was no syntax error. So... I don't konw...

**Comment by simonhaenisch** [comment]
I just released `1.0.1`, can you give it a try please and let me know if it happens again? Thanks!

**Comment by SUCHMOKUO** [comment]
OK. Thank for your working :)
